src/services/posts/createPost.js

```diff
@@ -0,0 +1,10 @@
+import axios from "axios"
+
+export const createPost = async ({postData}) => {
+    try {
+        const res = await axios.post(`/api/posts`, {postData});
+    }
+    catch (e) {
+
```

Comment:
changes: not there any code inside catch block everywhere

Reply:
It is left purposely for future use